### DISCUSSION TITLE
Need help,Enterprise server

### DISCUSSION THREAD

**Discussion by mlmbusiness83** [community/community#128590] [Enterprise]
### Body

Beginner, need help. github support takes to long or never responds. I downloaded github-enterprise-3.13.0.rc1.ova file.  the email from github don't tell you nothing or steps.  I doing this solo.

**Comment by queenofcorgis** [comment]
Hi @mlmbusiness83 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
